### PR TITLE
Laravel 6: Entrust replacement

### PR DESCRIPTION
**Pull request by eronisko**
Replacing https://github.com/Zizaco/entrust with Laravel Gates https://laravel.com/docs/6.x/authorization#gates

* user `role` is now defined as a column in users table
* Entrust tables (permission, role, etc) are only re-named (not deleted) for now. The migration would be difficult to rollback otherwise & I didn't want to delete them just in case. They'll be deleted in a separate migration.

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -2,24 +2,15 @@
...
-
-    use EntrustUserTrait;
-
-    /**
-     * The database table used by the model.
-     *
-     * @var string
-     */
-    protected $table = 'users';
+    public static $roles = ['admin', 'editor'];
```
Comment (igor-kamil) [inline app/User.php:10]: 'admin' or 'administer'? 
To be honest, I am in doubts, what is better. Whether "old" admin or new "administer" (what probably makes sensw for "role" as "ability").

File: config/cache.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'default' => env('CACHE_DRIVER', 'array'), //temp fix for entrust - https://github.com/Zizaco/entrust/issues/541 - should use redis or memcached
+    'default' => env('CACHE_DRIVER', 'file'),
```
Comment (igor-kamil) [inline config/cache.php:16]: Temp for long long time. Finaly fixed! ;)

**Review by igor-kamil — APPROVED** [review]
I confirm. This gate looks much better, then entrust. Looks fine

File: app/User.php (hunk 1 above)
Comment (eronisko) [reply app/User.php:10]: 'admin' is a role (noun), 'administrate' is an weirdly sounding verb -- the ability of an admin. I totally get why it looks off but I thought 'administrate' would also be easier to search for if we do some refactoring later.

**Pull request merged**